code/controllers/subsystem/garbage.dm

```diff
@@ -324,79 +345,86 @@ SUBSYSTEM_DEF(garbage)
 	var/no_hint = 0 //!Number of times it's not even bother to give a qdel hint
 	var/slept_destroy = 0 //!Number of times it's slept in its destroy
 	var/qdel_flags = 0 //!Flags related to this type's trip thru qdel.
+	var/list/extra_details //!Lazylist of string metadata about the deleted objects
 
 /datum/qdel_item/New(mytype)
 	name = "[mytype]"
 
-
 /// Should be treated as a replacement for the 'del' keyword.
 ///
 /// Datums passed to this will be given a chance to clean up references to allow the GC to collect them.
-/proc/qdel(datum/D, force=FALSE, ...)
-	if(!istype(D))
-		del(D)
+/proc/qdel(datum/to_delete, force = FALSE)
+	if(!istype(to_delete))
+		DREAMLUAU_CLEAR_REF_USERDATA(to_delete)
```

Comment:
This is the only time Dreamluau is ever called in the code outside of the bindings file. Again, this *only* lets Dreamluau check if it has any references to the object in Lua code, nothing else.

To remove Dreamluau so that this PR stands on its own, just get rid of this single line of code and the _dreamluau.dm file, along with the scripts in tools.